<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAssoc</class>
 <widget class="QDialog" name="DialogAssoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="4" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBoxRTM">
     <property name="title">
      <string>RTM</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelSignificanceLevel">
        <property name="text">
         <string>Significance level:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditSignificanceLevel">
        <property name="text">
         <string>0.05</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPreselection">
        <property name="text">
         <string>Pre-selection:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPreselection">
        <property name="text">
         <string>0.05</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelMaxNumQTLs">
        <property name="text">
         <string>Max. number QTLs:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditMaxNumQTLs"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelMaxModelRSquare">
        <property name="text">
         <string>Max. model R-Square:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditMaxModelRSquare">
        <property name="text">
         <string>0.99</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="labelMultTest">
        <property name="text">
         <string>Multiple testing:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="comboBoxMultTest">
        <item>
         <property name="text">
          <string/>
         </property>
        </item>
        <item>
         <property name="text">
          <string>BON</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>FDR</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="labelSumSqrType">
        <property name="text">
         <string>Sum of squares type:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboBoxSumSqrType">
        <item>
         <property name="text">
          <string>I</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>III</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBoxData">
     <property name="title">
      <string>Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QComboBox" name="comboBoxGenotype">
        <item>
         <property name="text">
          <string>VCF</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>PED/MAP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HapMap</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Genotype</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditGenotype"/>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="pushButtonGenotype">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelPhenotype">
        <property name="text">
         <string>Phenotype:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditPhenotype"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonPhenotype">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelCovariate">
        <property name="text">
         <string>Covariate:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEditCovariate"/>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonCovariate">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labelKinship">
        <property name="text">
         <string>Kinship:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="lineEditKinship"/>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButtonKinship">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBoxOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="labelMethod">
        <property name="text">
         <string>Analysis method:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxMethod">
        <item>
         <property name="text">
          <string>RTM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LM</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LMM</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="checkBoxGxE">
        <property name="text">
         <string>Gene-environment interaction</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxGenotype</tabstop>
  <tabstop>lineEditGenotype</tabstop>
  <tabstop>pushButtonGenotype</tabstop>
  <tabstop>lineEditPhenotype</tabstop>
  <tabstop>pushButtonPhenotype</tabstop>
  <tabstop>lineEditCovariate</tabstop>
  <tabstop>pushButtonCovariate</tabstop>
  <tabstop>lineEditKinship</tabstop>
  <tabstop>pushButtonKinship</tabstop>
  <tabstop>lineEditSignificanceLevel</tabstop>
  <tabstop>lineEditPreselection</tabstop>
  <tabstop>lineEditMaxNumQTLs</tabstop>
  <tabstop>lineEditMaxModelRSquare</tabstop>
  <tabstop>comboBoxMultTest</tabstop>
  <tabstop>comboBoxSumSqrType</tabstop>
  <tabstop>comboBoxMethod</tabstop>
  <tabstop>checkBoxGxE</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAssoc</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAssoc</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
